package.json

```diff
@@ -82,7 +83,7 @@
     "gulp-typescript": "^3.1.6",
     "gulp-watch": "^4.3.11",
     "husky": "^0.13.4",
-    "jest": "^21.1.1",
+    "jest": "<22",
```

Comment:
Starting with Jest v22, this errors occurs in Storybook:
* storybooks/storybook#2509
* storybooks/storybook#2522
  (this ticket mentions TS, but it’s not related to that)